sstables_loader.hh

```diff
@@ -134,3 +135,6 @@ public:
 
     class download_task_impl;
 };
+
+using shared_sstables = std::vector<sstables::shared_sstable>;
```

Comment:
I'd leave it as `std::vector<sstables::shared_sstable>`.

We have a ton of those already, and if at all, I'd convert all uses to a new type.
And if doing that, to distinguish this against:
```
sstables/shared_sstable.hh:39:using sstable_list = std::unordered_set<shared_sstable>;
```

I'd define in sstables/shared_sstable.hh
```
using sstable_vector = std::vector<shared_sstable>;
```

Reply:
One reason not to do those cleanups within the scope of this PR is backportability.